config/vufind/EDS.ini

```diff
@@ -335,6 +335,17 @@ AU = None
 ; This repeatable setting can be used to filter out values using group codes:
 ;excludeGroup[] = 'URL'
 
+; This section can be used to order items based on their labels.
+[ItemCoreOrder]
+; Note the label of the item field from the EDS API first then the relative order it should be shown in
+;Title         = 0
+;Authors       = 1
+;Source        = 2
+;Abstract      = 3
+;Subject Terms = 4
+
+
```

Comment:
Style: I don't think we need three blank lines between sections; one is probably enough.

Reply:
Fixed up in my follow-up commit